sidebars.js

```diff
@@ -33,10 +33,10 @@ const sidebars = {
       ],
     },
     {
-      "Setting Up Your Environmant": ["intro/environment-setup/stub"],
+      "Setting Up Your Environment": ["intro/environment-setup/overview", "intro/environment-setup/local_installation", "intro/environment-setup/tools_via_docker"],
```

Comment:
Two thoughts here…
1. We might not need to list all of these explicitly if we leverage `autogenerate` as some other sections do. (That does have some other implications for ordering, though.)
2. It looks like your editor isn't set up to enforce our prettier config, which could make future diffs messy.

We should think about how we want to standardize on both of these fronts. @eak12913 @oredavids Your technical insights here would be helpful.

Reply:
I was actually just about to comment that we should use `autogenerate` blocks here. Seems like that's preferable to use whenever it's possible.

Regarding the auto linting here are some ideas:
- We need to publish instructions for setting this up in various READMEs
- We could create a pre-commit hook that will just do this for you and not enforce anything?
- Similar idea as the hook - but reject commits that haven't been formatted?

Reply:
All of those sound like good ideas to me.

I'm on board for switching to auto-generation, but I propose we tackle it in a follow-up PR. It isn't required to get the MVP out the door even if it is the best practice we want to establish going forward. The explicit sidebar is carryover from early explorations in how to use Docusaurus, so it's just tech debt we'll need to clean up.

Reply:
I've tracked this in https://github.com/gruntwork-io/docs/issues/109